### PR TITLE
Add parameters' type information in JSDoc comments scaffolding

### PR DESCRIPTION
**Pull request by anubmat**
Adds type {any} in comments for parameters in JS and JSX files
image

Earlier type was missing:
image

### CODE REVIEW

**Comment by anubmat** [comment]
@jramsay , @billti , @vlad, Can someone please review?

File: src/services/jsDoc.ts [hunk 1]
```diff
@@ -166,15 +166,20 @@ namespace ts.JsDoc {
         const lineStart = sourceFile.getLineStarts()[posLineAndChar.line];
 
         const indentationStr = sourceFile.text.substr(lineStart, posLineAndChar.character);
+        const isJavaScriptFile = hasJavaScriptFileExtension(sourceFile.fileName); // JSX or JS file
```
Comment (DanielRosenwasser) [inline src/services/jsDoc.ts]: I don't think that this comment adds anything.

**Comment by DanielRosenwasser** [comment]
👍 after addressing my one nit.

File: src/services/jsDoc.ts (hunk 1 above)
Comment (anubmat) [inline src/services/jsDoc.ts]: agreed.. added because of jsx files... will remove.

**Comment by billti** [comment]
Looks good for now. We might want to make this behavior optional (based on a editor config setting) if we get feedback that some folks don't like it. I'd assume some devs don't want the type annotations in their JsDoc comments. Thanks! 👍

**Pull request merged**